### PR TITLE
release announcement + homepage updates

### PR DESCRIPTION
**Pull request by sfc-gh-jreini**
# Description

- Add 1.0 blog to trulens.org
- add banner announcement pointing to blog
- update images and copy on homepage.

## Other details good to know for developers

Please include any other details of this change useful for _TruLens_ developers.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
  not work as expected)
- [ ] New Tests
- [ ] This change includes re-generated golden test results
- [x] This change requires a documentation update

### CODE REVIEW

File: docs/trulens/release_blog_1dot.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+* Minimize the overhead required for TruLens developers to use the capabilities they need
+* Make it easy to understand what code and set of dependencies is needed
+
+## A note on versioning and backwards compatibility:
+
+Today, we’re releasing trulens, trulens-core, trulens-dashboard, trulens-feedback, trulens-providers packages and trulens-instrument packages at v1.0. We will not make breaking changes in the future without bumping the major version.
+
+The base install of trulens will install trulens-core, trulens-feedback and trulens-dashboard making it easy for developers to try TruLens.
+
+trulens-eval will remain at major version 0. All existing trulens-eval imports are mapped to v1 packages, and v1 packages are installable with trulens-eval.
```
Comment (sfc-gh-pmardziel) [inline docs/trulens/release_blog_1dot.md:29]: This bit is confusing. Perhaps rephrase: `trulens-eval` starting 1.0.0 will consist solely of backwards compatibility modules that alias `trulens-*` packages while emitting deprecation warnings. `trulens-eval` version will track `trulens-core` version for 3 months after which the deprecation warnings will be replaced by error messages instructing the user to update to `trulens`.

Comment (sfc-gh-jreini) [reply docs/trulens/release_blog_1dot.md]: updated

File: docs/assets/images/trulens_1_release_graphic_modular.png
Comment (sfc-gh-pmardziel) [inline docs/assets/images/trulens_1_release_graphic_modular.png:1]: `trulens-instrument-*` -> `trulens-apps-*`.

File: docs/trulens/release_blog_1dot.md
Comment (sfc-gh-pmardziel) [inline docs/trulens/release_blog_1dot.md:1]: `trulens-instrument` -> `trulens-app` in many places here.

File: docs/trulens/release_blog_1dot.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+You can see how to start a TruLens session logging to a postgres database below:
+
+!!! example "Start a TruLens Session"
+
+    ```python
+    from trulens.core import TruSession
+    from trulens.core.database.connector import DefaultDBConnector
+
+    connector = DefaultDBConnector(database_url="postgresql://trulensuser:password@localhost/trulens")
+    session = TruSession(connector=connector)
```
Comment (sfc-gh-pmardziel) [inline docs/trulens/release_blog_1dot.md:265]: Make note that `TruSession(database_url=...)` does the same thing as shown.

File: docs/trulens/release_blog_1dot.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+
+## **Versioning and Backwards Compatibility**
+
+Today, we’re releasing trulens, trulens-core, trulens-dashboard, trulens-feedback, trulens-providers packages and trulens-instrument packages at v1.0. We will not make breaking changes in the future without bumping the major version.
+
+The base install of trulens will install trulens-core, trulens-feedback and trulens-dashboard making it easy for developers to try TruLens.
+
+trulens-eval starting 1.0.0 will consist solely of backwards compatibility modules that alias trulens-* packages while emitting deprecation warnings. trulens-eval version will track trulens-core version for 6 weeks (Until October 10, 2024) after which the deprecation warnings will be replaced by error messages instructing the user to update to trulens.
+
+During the depreciation period, trulens-eval will track the trulens version and contain aliases for non-private API elements while giving deprecation warnings upon usage. After the depreciation period, the latest version of trulens_eval will refuse to install with a note that it has been deprecated.
```
Comment (sfc-gh-pmardziel) [inline docs/trulens/release_blog_1dot.md]: Redundant paragraph, delete.

File: docs/trulens/release_blog_1dot.md
Comment (sfc-gh-pmardziel) [reply docs/trulens/release_blog_1dot.md:1]: Many code terms not formatted as code with backticks in this file.

**Review by sfc-gh-pmardziel — COMMENTED** [review]
See individual comments.

File: docs/assets/images/trulens_1_release_graphic_modular.png
Comment (sfc-gh-jreini) [reply docs/assets/images/trulens_1_release_graphic_modular.png:1]: Asked designer to update

Comment (sfc-gh-chu) [inline docs/assets/images/trulens_1_release_graphic_modular.png:1]: missing trulens-connectors-*

File: docs/trulens/release_blog_1dot.md [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+
+In our suggested workflow, `app_name` represents an objective you’re building your LLM app to solve. All apps with the same `app_name` should be directly comparable with each other. Then `app_version` can be used to track each experiment. This should be changed each time you change your application configuration. To more explicitly track the changes to individual configurations and semantic names for versions - you can still use app `metadata` and `tags`!
+
+!!! example "Track Experiments"
+
+    ```python
+    tru_rag = TruCustomApp(
+    rag,
+    app_name="RAG",
+    app_vesion="v1",
```
Comment (sfc-gh-chu) [inline docs/trulens/release_blog_1dot.md:283]: *version

File: docs/trulens/release_blog_1dot.md [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+    tags="prototype",
+    metadata=metadata={
+                "top_k": top_k,
+                "chunk_size": chunk_size,
+            }
+    )
+    ```
+
+To bring these changes to life, we've also added new filters to the Leaderboard and Evaluations pages. These filters give you the power to focus in on particular apps and versons, or even slice to apps with a specific tag or metadata.
```
Comment (sfc-gh-chu) [inline docs/trulens/release_blog_1dot.md:292]: *version

File: docs/trulens/release_blog_1dot.md [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+
+The base install of trulens will install `trulens-core`, `trulens-feedback` and `trulens-dashboard` making it easy for developers to try _TruLens_.
+
+Starting 1.0.0, the `trulens_eval` package is being deprecated in favor of `trulens` and several associated required and optional packages.
+
+Until 10-14-2024, backwards compatibility during the warning period is provided by the new content of the `trulens_eval` package which provides aliases to the in their new locations. See [trulens_eval](./api/trulens_eval/index.md).
+
+Starting 10-15-2014 until 2025-12-01. Usage of `trulens_eval` will produce errors indicating deprecation.
+
+Beginning 2024-12-01 Installation of the latest version of `trulens_eval` will be an error itself with a message that `trulens_eval` is no longer maintained.
```
Comment (sfc-gh-chu) [inline docs/trulens/release_blog_1dot.md:36]: inconsistent date schemes

File: docs/trulens/release_blog_1dot.md [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+
+## **Split off trulens-eval from trulens-explain**
+
+Split off `trulens-eval` from `trulens-explain`, and let `trulens-eval` take over the `trulens` package name. _TruLens-Eval_ is now renamed to _TruLens_ and sits at the root of the _TruLens_ repo, while _TruLens-Explain_ has been moved to its own repository, and is installable at `trulens-explain`.
+
+![TruLens 1.0 Release Graphics](../assets/images/trulens_1_release_graphic_split.png)
+
+## **Separate TruLens-Eval into different trulens packages**
+
+Next, we modularize _TruLens_ into a family of different packages, described below. This change is designed to minimize the overhead required for _TruLens_ developers to use the capabilities they need. For example, you can now install instrumentation packages in production without the additional dependencies required to run the dashboard.
```
Comment (sfc-gh-chu) [inline docs/trulens/release_blog_1dot.md:14]: nit: modularized

File: docs/trulens/release_blog_1dot.md [hunk 8]
```diff
@@ -0,0 +1,353 @@
...
+
+* `trulens-core` holds core abstractions for database operations, app instrumentation, guardrails and evaluation
```
Comment (sfc-gh-chu) [inline docs/trulens/release_blog_1dot.md:16]: nit: missing punctuation

File: docs/assets/images/trulens_1_release_graphic_modular.png
Comment (sfc-gh-jreini) [reply docs/assets/images/trulens_1_release_graphic_modular.png:1]: Changed to apps, but will have to ship without connectors. Too late to get that in.

**Pull request merged**